macOnly.js

```diff
@@ -1,46 +1,13 @@
 const { ipcMain } = require('electron')
 
-function doubleClick (win) {
-  let smallPosition = win.getBounds()
-  let bigPosition
-  const customSize = 300
-  const bigFactor = 2 // how much enlarge when double click video
-  const proportion = customSize * bigFactor
+function changeWindowWithDoubleClick (screenController) {
```

Comment:
I think we should name this as `toggleWindowSize` and put this inside screenController, so even if we only use this on MacOS we can have the function in the right place.

Reply:
I think you're right 🥰